docs/guides/access-groups.mdx

```diff
@@ -105,45 +139,58 @@ When you create this access group, include the following policies.  You can copy
 }
 ```
 
+- All IAM Account Management services (administrator)
+
+
 <span id="ui"></span>
-## Create an access group and invite users to an instance (user interface)
+### Use the IBM Cloud IAM console
 
-As an account owner or administrator, you add the people in your organization to the IBM Cloud account and then assign them access permissions to use an instance.
+As an account owner or administrator, follow these steps to create an IBM Quantum Administrator access group.
 
 1. Go to Manage > [Access (IAM)](https://cloud.ibm.com/iam/overview) in the IBM Cloud console.
 1. On the left panel in the **Manage access** section, click **Access groups**, then click **Create**.
-1. In the **Create access group** window that opens, add a name and description that represent the group of users that you will invite.  For example, the company name. Click **Create**.
-1. In the access group just created, click the **Access** tab, then click **Assign access** at the top right.
-1. In the **Create policy** page that opens, define these elements. Alternatively, you can copy and paste code into the JSON entry feature, as described in the [previous section.](#json)
-   - **Service** - In the search field look for Qiskit Runtime and select it. Click **Next**.
+1. In the **Create access group** window that opens, add a name and description that represent the group of users that you will invite.  For example, IBM Quantum Administrators. Click **Create**.
+
+Next, create policies for the Qiskit Runtime service,  for All IAM Account Management services, and for All Account Management services.
+1. In the access group just created, click the **Access** tab, then click **Assign access**.
+1. In the **Create policy** page that opens, define these elements:
+   - **Service** - Search for Qiskit Runtime and select it. Click **Next**.
    - **Resources** - Select **Specific resources**, and in the rest of the dropdowns, choose the following:
       - For **Attribute type**, select **Service Instance**.
-      - For **Operator**, select **string equals**.
+      - For the operator, leave **string equals**.
       - For **Value**, select the instance you want to grant access to. Click **Next**.
```

Comment:
change this to just:
- **Resources** - Select **All resources** and click **Next**.